v1/announcements.yaml

```diff
@@ -8,7 +8,7 @@ paths:
       description: |
         Gets announcements for display in the official Wikipedia iOS and Android apps.
 
-        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Unstable)
```

Comment:
I'd probably leave this one at experimental for now. It's unclear if or when the apps would move to using centralnotice instead. This endpoint was meant for the time when FR-tech was busy at the end of last year.